ui/src/app/richskill/list/skills-list.component.ts

```diff
@@ -17,6 +17,7 @@ import {ButtonAction} from "../../auth/auth-roles";
 import {CollectionService} from "../../collection/service/collection.service"
 import {ApiCollection} from "../../collection/ApiCollection"
 import {CollectionPipe} from "../../pipes"
+import {query} from "@angular/animations"
```

Comment:
@manuel-delvillar Are we sure that @angular/animations is the right import for this?

Reply:
It was an unused import, It was removed